includes/admin/class-wc-rest-payments-payment-intents-controller.php

```diff
@@ -34,6 +42,26 @@ public function register_routes() {
 				'permission_callback' => [ $this, 'check_permission' ],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			[
+				'methods'             => WP_REST_Server::CREATABLE,
+				'callback'            => [ $this, 'post_payment_intent' ],
+				'permission_callback' => [ $this, 'check_permission' ],
```

Comment:
I do know that this is POC, but if we are planning to add it to the production, could we use WordPress REST API Schemas and perhaps add validation like we do in reports API endpoints?

Reply:
That would require to either change all instances (including the current payment processing) into that format, or create a separate code for this instance, both of which does not seem advisable considering the area (charging a payment)